publify_core/app/views/archives_sidebar/_content.html.erb

```diff
@@ -1,6 +1,6 @@
 <% unless sidebar.archives.blank? %>
-  <h3 class="sidebar_title"><%= sidebar.title %></h3>
-  <div class="sidebar_body">
+  <h3 class="sidebar-title"><%= sidebar.title %></h3>
```

Comment:
oh no! you've included the commits from the other bug you fixed! you need to use branches more effectively!